routes/root.js

```diff
@@ -25,6 +27,321 @@ const router = express.Router();
 
 router.use(csrfProtection);
 
+const fetchLands = {
```

Comment:
Same comment about putting in a helper.

Reply:
this is a temporary function for the database migration